chia/wallet/wallet_node.py

```diff
@@ -86,6 +86,12 @@ def get_wallet_db_path(root_path: Path, config: Dict[str, Any], key_fingerprint:
     return path
 
 
+@dataclasses.dataclass
```

Comment:
I'd default to this at least when there are immutable attribute types and we don't need to mutate.

```suggestion
@dataclasses.dataclass(frozen=True)
```